webpack.config.js

```diff
@@ -127,7 +127,12 @@ var wpconfig = {
     },
 
     externals: {
-        gettext: 'gettext'
+        backbone: 'Backbone',
```

Comment:
Good catch. This list should be the same. Note that it currently works because we don't have code bundled by Webpack that depends upon the more esoteric libraries, e.g. Mathjax. 

I'll make them the same.